gammapy/modeling/models/core.py

```diff
@@ -612,6 +669,61 @@ def selection_mask(
 
         return np.array(selection, dtype=bool)
 
+    def contribute(self, mask, psf=None):
```

Comment:
I would argue taking a `PSFKernel` object here is non-ideal, as it's rather complex to create. A simple number like `margin="0.6 deg"` is in general sufficient. So having a signature like:

```python3
models.select_mask(mask, margin="0.6 deg", inside=True)
```
Is easier to use...the connection to the `PSFKernel` could be done using an example in the doctoring instead.